src/Location.vue

```diff
@@ -72,7 +72,7 @@ export default {
 				oldElm.style.fill = null
 			}
 			if (newElm) {
-				newElm.style.fill = '#e6605c'
+				newElm.style.fill = 'var(--color-primary)'
```

Comment:
@skjnldsv Can we use css vars just like that inside a vue component?

Reply:
Should work yes :)